zoomus/components/live_stream_status.py

```diff
@@ -0,0 +1,21 @@
+from __future__ import absolute_import
+
+from zoomus import util
+from zoomus.components import base
+
+
+class LiveStreamStatusComponentV2(base.BaseComponent):
```

Comment:
I know that the status is technically separate endpoint on Zoom's side. but do you think for the sake of simplicity in this library that getting a status should be part of the live_stream component. I.e. I would think that doing
`zoom.live_stream.update` and `zoom.live_stream.update_status` might be cleaner/easier? Thoughts?

Reply:
Hi Patrick, I think It actually makes sense. I think the logic applies to the same element (A Live Stream) and the only difference is the payload and the endpoint. Let me check, I will see what I can do. 

Please don't update the develop branch again 🤪

Reply:
Hi! Great! 

I just bumped one library dependency, so please do merge develop in again, but it should be transparent to this work. I'll promise not to make any more merges to develop until this gets merged. in. 

Thank you again!

Reply:
Hi Patrick! Changes are done. Just waiting for the merge :) Cheers!